apps/authentication-service/src/shared/http/httpService.ts

```diff
@@ -0,0 +1,142 @@
+import { ResponseError } from "auth0";
```

Comment:
This is a really complex file. What purpose is it solving?

Reply:
Service removed.